src/content/en/updates/2016/12/imagecapture.md

```diff
@@ -0,0 +1,329 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: ImageCapture is an API for controlling camera settings and taking photos.
+
+{# wf_updated_on: 2016-12-05 #}
+{# wf_published_on: 2016-12-05 #}
+{# wf_tags: canvas,chrome56,media,webrtc #}
+{# wf_featured_image: /web/updates/images/2016/12/imagecapture/featured.jpg #}
+{# wf_featured_snippet: ImageCapture is an API for controlling camera settings and taking photos. #}
+
+<style>
+#ic-demo .hidden {
+display: none;
+}
+
+#ic-demo button {
+display: block;
+float: left;
+margin: 0 10px 20px 0;
+width: 120px;
+}
+
+#ic-demo button:last-of-type {
+float: none;
+margin: 0 0 20px 0;
+}
+
+#ic-demo canvas {
+display: block;
+margin: 0 0 20px 0;
+max-width: 100%;
+}
+
+#ic-demo img {
+display: block;
+margin: 0 0 20px 0;
+}
+
+#ic-demo input#zoom {
+margin: 0 0 20px 0;
+width: 100%;
+}
+
+#ic-demo select {
+margin: 0 0 20px 0;
+}
+
+#ic-demo video {
+margin: 0 0 20px 0;
+vertical-align: top;
+max-width: 100%;
+}
+
+</style>
+
+# Take Photos & Control Camera Settings {: .page-title }
+
+{% include "web/_shared/contributors/samdutton.html" %}
+
+`ImageCapture` is an API to capture still images and control camera settings.
+
+The API enables control over camera hardware features such as zoom, focus mode,
+contrast, ISO and white balance. Best of all, `ImageCapture` enables access to the full resolution capabilities. Previous techniques for
+taking photos on the Web have used snapshots from a video stream, which are
+lower resolution than those available for still images.
+
+`ImageCapture` is available in Chrome 56 on desktop and Android as an
+[Origin Trial](https://github.com/jpchase/OriginTrials/blob/gh-pages/developer-guide.md),
+or in Chrome Canary on desktop and Android with **Experimental Web Platform**
```

Comment:
Can we "link" directly to `chrome://flags/#enable-experimental-web-platform-features`?